packages/components/pie-button/playwright-lit.config.ts

```diff
@@ -0,0 +1,52 @@
+import { defineConfig, devices } from '@sand4rt/experimental-ct-web';
```

Comment:
Eventually I'd like to move all the shared thing from the various playwright configs into a single file, but just wanted to prove components can be tested as part of this PR.